mcx/sample.py

```diff
@@ -507,8 +527,14 @@ def sample_loop(
         The parameters of the evaluator.
     rng_keys: array (n_samples,)
         JAX PRNGKeys used for each sampling step.
-    num_chains
+    num_chains : int
         The number of chains
+    call_backs:
+        The functions to run after each state update
```

Comment:
This does not appear in the function's signature

Reply:
Oops. Fixed! I'll also squash all the commits so it's ready to merge in